### ISSUE TITLE
turf.circle units typing is wrong

### ISSUE THREAD

**Issue by bradgreens** [Turfjs/turf#2391]
- [x] "@turf/circle": "^6.5.0"
- [x] GeoJSON data / source code

```
  circle([-68.53931, 45.06519], 0.5, {
    units: 'acres' // typescript autocompletes this, but it is an error
  });
```

The typings for the units option of a circle are wrong. The type is set to: `units?: Units;`. This maps to the full `Units` type `export declare type Units = "meters" | "millimeters" | "centimeters" | "kilometers" | "acres" | "miles" | "nauticalmiles" | "inches" | "yards" | "feet" | "radians" | "degrees" | "hectares";`;

However, [the documentation](https://github.com/Turfjs/turf/tree/master/packages/turf-circle#circle) shows that only `degrees, radians, miles, or kilometers` are allowed. This is confusing because my editor allowed me to autocomplete `acres` due to the misleading type definition.

![Screenshot 2023-01-23 at 12 31 59 PM](https://user-images.githubusercontent.com/284366/214147048-2160d2dd-6b95-4db2-9290-897855a99d69.png)

❤️ Turf. Thank you!

**Comment by bradgreens** [comment]
This issue is relevant to many packages. e.g. @turf/distance and many more.

**Comment by bradgreens** [comment]
Added PR https://github.com/Turfjs/turf/pull/2392

**Comment by JamesLMilner** [comment]
Hi @bradgreens  - thanks for raising this. I have looked into this a little more based off your prompt. It looks like `turf.circle` passes through the units option to `turf.destination`, which then uses it in ` lengthToRadians(distance, options.units);` This function indeed is using the `Units` type which includes `hectares` and `acres` which are exclusively measurements of area. 

I have raised a PR which aims to fix this by seperating out `AreaUnits` from regular `Units` which should fix this issue.
